From: Keith Packard <keithp@keithp.com>
Date: Wed, 2 Sep 2020 10:22:52 -0700
Subject: Use <stdbool.h> instead of "config.h" in published header file

We don't want to publish our "config.h" as that name is rather too
generic to put in /usr/include. Instead, assume the compiler has
stdbool.h, which debian systems should always have.

Closes: #969212.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
 extensions/cmark-gfm-core-extensions.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extensions/cmark-gfm-core-extensions.h b/extensions/cmark-gfm-core-extensions.h
index 0645915..3a32f76 100644
--- a/extensions/cmark-gfm-core-extensions.h
+++ b/extensions/cmark-gfm-core-extensions.h
@@ -7,7 +7,7 @@ extern "C" {
 
 #include "cmark-gfm-extension_api.h"
 #include "cmark-gfm-extensions_export.h"
-#include "config.h" // for bool
+#include <stdbool.h>
 #include <stdint.h>
 
 CMARK_GFM_EXTENSIONS_EXPORT
